app/controllers/publishers/ats_api/v1/update_vacancy_service.rb

```diff
@@ -0,0 +1,42 @@
+module Publishers
+  module AtsApi
+    module V1
+      class UpdateVacancyService
+        class << self
+          def call(vacancy, params)
+            if vacancy.update(permitted_params(params))
+              { success: true }
+            else
+              { success: false, errors: format_errors(vacancy.errors.messages) }
+            end
+          end
+
+          private
+
+          attr_reader :vacancy, :params
+
+          def permitted_params(params)
```

Comment:
I found this method name confusing given that the controller does the 'permitted params' filtering